scripts/config.json

```diff
@@ -15,6 +15,21 @@
       "mounts": {
         "service": "/data/service"
       }
+    },
+    {
+      "documentation": {
+        "command": "install",
+        "description": "installs combined node modules of service and template",
+        "arguments": []
+      },
+      "environment": {
+        "image": "semtech/mu-javascript-template:2.0.0",
```

Comment:
this has to be the version of the js image as will be published (feature branch?), if you use e.g. 1.8.0, it will not have the install scripts that are required.